src/Plugin/resource/Field/ResourceFieldEntity.php

```diff
@@ -207,6 +242,11 @@ public function access($op, DataInterpreterInterface $interpreter) {
 
     // Perform field API access checks.
     $property_wrapper = $this->propertyWrapper($interpreter);
+    if ($this->isWrapperMethodOnEntity() && $this->getWrapperMethod() && $this->getProperty()) {
+      // Sometimes we define fields as $wrapper->getIdentifier. We need to
+      // resolve that to $wrapper->nid to call $wrapper->nid->info().
+      $property_wrapper = $property_wrapper->{$this->getProperty()};
```

Comment:
Sure.

This is trying to get the property that is behind `EntityDrupalWrapper` helper methods. For instance, for a node it will return `nid` if we are using `getIdentifier()` and `fid` for a file entity.

The idea behind this is that if we have a node resource with:

```php
array(
  'title' => array(
    'method_wrapper' => 'label',
    'method_on_wrapper' => TRUE, // Don't remember the exact key.
  ),
);
```

We can read without problem, but we won't be able to map that back to the `title` column in the `node` table. What this call does is return `title` for the `label` callback.